weld/subsystem/src/main/java/org/jboss/as/weld/services/bootstrap/ProxyServicesImpl.java

```diff
@@ -106,7 +115,7 @@ private ClassLoader _getClassLoader(Class<?> proxiedBeanType) {
                     logWarning = processedStaticModules.putIfAbsent(definingModule.getIdentifier(), hasWeldDependencies) == null;
                 }
                 if (hasWeldDependencies) {
-                    // this module declares weld dependencies - we can use module's classloader to load the proxy class
+                    // this modulew declares weld dependencies - we can use module's classloader to load the proxy class
```

Comment:
Is this an intentional change or a typo?

Reply:
That's just a typo